cmd/nerdctl/image/image_list.go

```diff
@@ -70,49 +70,47 @@ Properties:
 	return imagesCommand
 }
 
-func processImageListOptions(cmd *cobra.Command, args []string) (types.ImageListOptions, error) {
+func processImageListOptions(cmd *cobra.Command, args []string) (*types.ImageListOptions, error) {
```

Comment:
Change the signature here so that we can return nil instead of `types.ImageListOptions{}`.